src/ol/Overlay.js

```diff
@@ -378,7 +378,7 @@ class Overlay extends BaseObject {
 
   /**
    * Set the map to be associated with this overlay.
-   * @param {import("./PluggableMap.js").default|undefined} map The map that the
+   * @param {import("./PluggableMap.js").default|undefined|null} map The map that the
```

Comment:
Preferably we remove undefined here to keep it consistent but that would be a breaking change so I didn't do it yet.

Reply:
Feel free to remove it now, and also make the argument mandatory where it is currently optional. It's just type improvements.